packages/eslint-plugin-turbo/lib/rules/no-undeclared-env-vars.ts

```diff
@@ -81,6 +172,17 @@ function create(context: RuleContextWithOptions): Rule.RuleListener {
     }
   });
 
+  const filename = context.getFilename();
+  debug(`Checking file: ${filename}`);
+
+  const matches = frameworkEnvMatches(filename);
```

Comment:
There's a lot of things in here that _could_ go wrong. Reading files / parsing JSON etc. We should probably try / catch this to be as defensive as we can, and if something goes wrong, log it - but continue.

Reply:
good call: upon re-inspection for this it was already fairly close, but there were two areas that needed attention.  addressed in 6a3807333acce67cd253ea5d258de285ba10852b